groups/nts/ntsu/ntsu_socketutil.cpp

```diff
@@ -7788,6 +7788,23 @@ bsl::size_t SocketUtil::maxBacklog()
 
 bool SocketUtil::isSocket(ntsa::Handle socket)
```

Comment:
By the way, I did not find any test for this function. I think it's relatively easy to test it, so let's do it.

Reply:
I have added a test for ntsu::SocketUtil::isSocket.